stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -617,6 +617,10 @@ internal class ChannelControllerImpl(
                     val result = uploadAttachment(it, attachmentTransformer)
                     if (result.isSuccess) {
                         attachment = result.data()
+                    } else {
+                        attachment.name = result.data().name
```

Comment:
Here you can get an exception. Only `isSuccess` check guarantees that data isn't null.

Reply:
Makes sense.

Reply:
Removed the name.